pkgs/applications/science/math/speedcrunch/default.nix

```diff
@@ -19,6 +19,14 @@ mkDerivation rec {
     cd src
   '';
 
+  patches = [
+    (fetchpatch {
+      name = "breeze-colorschemes.patch";
```

Comment:
Can you put in a comment what this fix is for?

The 0.12 release is from 2016, but speedcrunch is being actively developed.  I think we should just switch to the git version.

Reply:
Bumping the whole thing to unstable seems sensible to me.